openapi/src/test/groovy/io/micronaut/openapi/visitor/OpenApiApplicationVisitorSpec.groovy

```diff
@@ -775,24 +770,12 @@ class MyBean {}
         oauth2.flows
         oauth2.flows.implicit
         oauth2.scheme == null
-
-        def withRef = openAPI.components.securitySchemes['schemeWithRef']
-        withRef.type == null
-        withRef.in == null
-        withRef.name == null
-        withRef.description == null
-        withRef.openIdConnectUrl == null
-        withRef.bearerFormat == null
-        withRef.flows == null
-        withRef.scheme == null
```

Comment:
Why was this deleted?

Reply:
@graemerocher Because the swagger doesn't really support links to other security schemes. Therefore, when you try to deserialize a swagger file with such a scheme, it will be ignored, because. type is not specified. See class SecuritySchemeDeserializer:
![изображение](https://github.com/micronaut-projects/micronaut-openapi/assets/48474279/4ad1d3be-8183-422c-80f9-522316af4f84)
That is, this test case is not correct.